getting-started/windows-calico/limitations.md

```diff
@@ -0,0 +1,118 @@
+|             | Application layer policy (ALP) for Istio                     |
+|             | Encryption with Wireguard                                    |
+| Networking  | Non-overlay mode with BGP peering                            |
+|             | IP in IP overlay with BPG routing                            |
+|             | Cross-subnet support and MTU setting for VXLAN               |
+|             | Service IP advertisement                                     |
+|             | IPv6 and dual stack                                          |
+
+### Networking limitations with {{site.prodname}} VXLAN 
+
+Because of differences between the Linux and Windows dataplane feature sets, the following {{site.prodname}} features are not supported on Windows.
+
+| Feature                  | Limitation                                                   |
+| ------------------------ | ------------------------------------------------------------ |
+| IPs reserved for Windows | {{site.prodname}} IPAM allocates IPs in CIDR blocks. Due to networking requirements on Windows, four IPs per Windows node-owned block must be reserved for internal purposes.<br /><br/>For example, with the default block size of /26, each block contains 64 IP addresses, 4 are reserved for Windows, leaving 60 for pod networking.<br /><br />To reduce the impact of these reservations, a larger block size can be configured at the IP pool scope (before any pods are created). |
+| Single IP block per host | {{site.prodname}} IPAM is designed to allocate blocks of IPs (default size /26) to hosts on demand. While the {{site.prodname}} CNI plugin was written to do the same, kube-proxy currently only supports a single IP block per host. <br />To allow multiple IPAM blocks per host (at the expense of kube-proxy compatibility), set the `windows_use_single_network` flag to `false` in the `cni.conf.template` before installing {{site.prodname}}. Changing that setting after pods are networked is not recommended because it may leak HNS endpoints. |
+| NAT-outgoing             | {{site.prodname}} IP pools support a "NAT outgoing" setting with the following behaviour: <br /><br />- Traffic between {{site.prodname}} workloads (in any IP pools) is not NATted. <br />- Traffic leaving the configured IP pools is NATted if the workload has an IP within an IP pool that has NAT outgoing enabled. {{ site.prodNameWindows }} honors the above setting but it is only applied at pod creation time. If the IP pool configuration is updated after a pod is created, the pod's traffic will continue to be NATted (or not) as before. NAT policy for newly-networked pods will honor the new configuration. {{ site.prodNameWindows }} automatically adds the host itself and its subnet to the NAT exclusion list. This behaviour can be disabled by setting flag `windows_disable_host_subnet_nat_exclusion` to `true` in `cni.conf.template` before running the install script. |
```

Comment:
We can now remove section about `NAT-outgoing` . I've merged a PR that Windows `NAT-outgoing` behaves exactly same as Linux.

Reply:
https://github.com/projectcalico/calico/pull/3851/commits/eb1ba2866c4acc985f0bf104a47056bf0cc5c739